core/lib/spree/core/controller_helpers/auth.rb

```diff
@@ -18,6 +18,7 @@ module Auth
         included do
           before_action :set_guest_token
           helper_method :try_spree_current_user
+          helper_method :spree_current_user
```

Comment:
The commit on solidus_auth_devise referenced in the commit message has a link that isn't valid; it should be `https://github.com/solidusio/solidus_auth_devise/commit/69269765`.

Reply:
Fixed, thanks for catching it!